src/propmcwf.jl

```diff
@@ -0,0 +1,92 @@
+immutable QuMCWF <: QuPropagatorMethod
+end
+
+type QuMCWFEnsemble{QA<:QuBase.AbstractQuArray}
+    ntraj::Int
+    rho::QA
+    decomp
+    QuMCWFEnsemble(ntraj, rho, decomp) = new(ntraj, rho, decomp)
+end
+
+QuMCWFEnsemble{QM<:QuBase.AbstractQuMatrix}(ntraj::Int, rho::QM) = QuMCWFEnsemble{QM}(ntraj, rho, eigfact(full(coeffs(rho))))
```

Comment:
If you turn the arguments around (`rho::QM, ntraj::Int`) you can use a default value for the latter and you don't need the extra constructors below.